### DISCUSSION TITLE
Marker error

### DISCUSSION THREAD

**Discussion by TekyTeky** [Novakasa/brickrail#150] [General]
Hi, Sorry, I'm not sure if this should be a new bug or part of an existing?
So I'll let you decide where to put it.
This is the first time I have run the trains for a while and when I ran in Random Mode the same City Train, got a Marker error in a common place.
There wasn't a pic file with todays date of what the sensor saw.
Also the BLE log file didn't have todays date... but there wasn't a BLE error except on startup.
I have attached a screen shot of the log file folder... so you can see the files.. let me know if you want any.

Also when I ran it in manual after it was in reverse, which was weird as it had moved there in forward so maybe it was about to reverse and got the error.

Where the train stopped and the normal logs
[brickrail_2023-07-20_16.29.56.log](https://github.com/Novakasa/brickrail/files/12102836/brickrail_2023-07-20_16.29.56.log)
[brickrail.log](https://github.com/Novakasa/brickrail/files/12102842/brickrail.log)
[Layout - 5.txt](https://github.com/Novakasa/brickrail/files/12102854/Layout.-.5.txt) ---- rename this to .brl

![Log file pic 20 July](https://github.com/Novakasa/brickrail/assets/87248517/b84a7ebf-f607-4932-9238-02c451c801a9)

![Marker Error 20July](https://github.com/Novakasa/brickrail/assets/87248517/2d54233a-614a-48d8-9a7a-deb3c0c7ecb5)

I just moved the Green train and it also got a marker error first time the next fwd block.
But I'm not sure it was caught on the log.
[brickrail.log](https://github.com/Novakasa/brickrail/files/12103120/brickrail.log)

I added another blue plate to that sensor and also increased Chroma to 4000 on each train and haven't had any more issues.

**Comment by Novakasa** [comment]
that's really weird that there is no sensor dump. According to the logs it was saved as `hub_CityTrain3_dump_2023-07-20_16.32.59.txt` and `hub_CityTrain3_colors_2023-07-20_16.32.59.png`. Maybe the ble-server logs would have more info on that, but apparently that wasn't written as well. Right now my guess is that the ble-server somehow missed the message that contained which path to write the files.

**Reply by Novakasa** [reply]
But, can you upload the other dump files in your directory? Would be interested to seem them, as they probably contain realistic data, and I haven't had the chance to test the dumps with a real layout. I am guessing those were marker errors that were due to user error?

**Reply by Novakasa** [reply]
Ah, interesting, the logs show that the first time the server didn't connect properly, but then you probably tried reconnecting it, which worked. But this is probably the reason the log path wasn't setup correctly.

Edit: tracked now in #152

**Reply by Novakasa** [reply]
I'm not sure whether you get notifications for @ mentions, so please head to #152, where I would like a bit more info
